.gitignore

```diff
@@ -5,3 +5,4 @@ composer.lock
 .DS_Store
 .php_cs.cache
 .hg
+.idea
```

Comment:
I wouldn't add this change since it's unrelated.

Reply:
Fixed